cypress/e2e/tests/migration/applicationinventory/applications/filter.test.ts

```diff
@@ -140,6 +141,16 @@ describe(["@tier3"], "Application inventory filter validations", function () {
         clickByText(button, clearAllFilters);
     });
 
+    it("Analysis filter validations",function(){
+        Application.open();
+        getFirstAnalysisColumnValue().then((firstValue) => {
+            cy.log("First Analysis column value: ", firstValue);
+            applySearchFilter(analysis,firstValue);
```

Comment:
This test suite creates 2 applications at the beginning and does not analyze it so we can be sure that at least there is going to be one application with the "Not started" status.

In fact, there is a high probability only apps with "Not started" status exist when this test is executed, so maybe it would be a good idea to create an app then start and cancel an analysis on it so we have, at least, 2 statuses to correctly test the feature.